packages/scandipwa/src/route/ProductPage/ProductPage.container.js

```diff
@@ -292,13 +292,13 @@ export class ProductPageContainer extends PureComponent {
     isProductInformationTabEmpty() {
         const dataSource = this.getDataSource();
 
-        return !dataSource?.description?.html.length;
+        return !!dataSource?.description?.html.length;
```

Comment:
Also, isEmpty should return true if the value is 0, isn't it? So this logic is wrong then?

Same below.